### DISCUSSION TITLE
Execute requests only on development or local NODE_ENV

### DISCUSSION THREAD

**Discussion by jonatandorozco** [permafrost-dev/node-ray#9] [Ideas]
Taking in account that Ray is a development tool, we don't need to send requests to ray when the code is on staging or production server.

The PHP version allow to keep the ray commands in code but just send the data on APP_ENV=local

It will be awesome to have the same feature

**Comment by patinthehat** [comment]
Agreed.  Can you open an issue for this?

**Reply by jonatandorozco** [reply]
Sure. I'll open one

**Reply by jonatandorozco** [reply]
Done. Let me know if I can help with something
